client/client.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/websocket"
+	"github.com/taskcluster/webhooktunnel/wsmux"
+)
+
+// Client is used to connect to a proxy and serve endpoints
+// defined in Handler
+type Client struct {
+	Id string
```

Comment:
proxyAddr should probably a property on the client too.... so we can easily reconnect....